File: hw_tutor_lesson_1.rb
# def ask_n
# 	puts "Give me a number"
# 	n = gets.chomp.to_i
# end

# def sum_all_num_1
# 	n = ask_n
# 	counter = 0
# 	sum = 0
# 	while counter <= n
# 		sum += counter
# 		counter += 1
# 	end

# 	return sum
# end


# #or

# def sum_all_num_2
# 	n = ask_n
# 	sum = 0

# 	(n+1).times do |x|
# 		sum = sum + x
# 	end

# 	return sum
# end


# puts sum_all_num_1
# puts sum_all_num_2


# Question 2 take three numbers and return the largest
def largest_num_1(x, y, z)
	if (x == x.to_i && y == y.to_i && z == z.to_i)
		if (x > y && x > z)
			return puts "x = #{x} is larger than y = #{y} and z = #{z}"
		elsif (y > x && y > z)
			return puts "x = #{y} is larger than x = #{x} and z = #{z}"
		else
			return puts "z = #{z} is larger than x = #{x} and y = #{y}"
		end
	end
end

puts largest_num_1(3, 4, 5)

File: hw_tutor_lesson_1.py
# Question 1 take a number and sum all the numbers that come after it 

def ask_for_num():
	n = int(raw_input('Please, give me a number'))
	return n

def sum_num_less_n():
	n = ask_for_num()
	ans = 0
	for x in range[:n+1]:
		ans += x
	return ans


# driver code
print sum_num_less_n



# Question 2 take three numbers and return the largest of the three

def largest_num_1():
	x = ask_for_num()
	y = ask_for_num()
	z = ask_for_num()
	if (x == int(x) and y == int(y) and z == int(z)):
		if (x > y and x > z):
			return print("x = "+  x + "is larger than y = "+ y  , "and z = "+  z)
		elif (y > x and y > z):
			return print("y = "+  y + "is larger than x = "+ x  + "and z = "+  z)
		else:
			return print("z = "+  z + "is larger than x = "+ x  + "and y = "+  y)
		



puts largest_num_1()
